lib/build.gradle

```diff
@@ -31,7 +31,16 @@ android {
 
 dependencies {
 
+    implementation 'androidx.annotation:annotation:1.4.0'
```

Comment:
Since we had the `implementation` block already, can we move this line inside of the block and maybe group it by the break line?